readme.md

```diff
@@ -102,6 +102,14 @@ We do not recommend using it on targets that are not URLs.
 
 Especially useful when dealing with the [double-quotes on Windows](#double-quotes-on-windows) caveat.
 
+##### allowNonzeroExitCode
+
+Type: `boolean`\
+Default: `false`
+
+Option to allow the opened app resolve the promise with nonzero exit code when `wait` option value `true`.
```

Comment:
The readme docs should be in sync with the index.d.ts. The text here is different.

Reply:
so sorry i miss the README